Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -44,6 +44,7 @@ $isPremiumStorage = $false
 
 $sourcePath = $sourcePath.Trim('"')
 $storageAccount = $storageAccount.Trim()
+$containerName = $containerName.Trim().ToLower()
```

Comment:
$containerName is non mandatory parameter so cant assume it is not null. Also we have already a code at line no 105 to pre-process this object, better to keep this at one place